app.js

```diff
@@ -23,6 +28,37 @@ app.use(function (req, res, next) {
     // Pass to next layer of middleware
     next();
 });
+//TODO create the client using the actual host when connected to dev or prod, do this by adding the host and port into the createclient() function
+
+
+
+//get and parse cookie and place it into req.cookies
+app.use(cookieParser());
+
+//TODO replace this with login route
+app.get("/testRoute/", function(req,res){
```

Comment:
I added a route that does not require a cookie to already be set that will give you a cookie. If you call this route you should then have the cookie to get to other routes saved in your browser.